File: Java/UberX.java
package Java;

// public class UberX {
    
// }

class UberX extends Car {
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver); // As if you called the constructor method of the car class and printDataCar method.
        this.brand = brand;
        this.model = model;
    }
}

// If we put super., we have access to the attributes of the parent class.
// So, SUPER will refer to the attributes and methods of the super class and THIS will refer to the attributes of the child class.

File: PHP/uberX.php
<?php
require_once("car.php"); //Bring PHP car file
class UberX extends Car { //Same syntax as Java
    public $brand;
    public $model;

    public function __construct($license, Account $driver, $brand, $model){
        parent::__construct($license, $driver);  //Equivalent to "super" in Java
        $this->brand = $brand; //New additional data
        $this->model = $model; //New additional data
    }

    public function printDataCar(){
        parent::PrintDataCar();
        echo "Brand: $this->brand
             Model: $this->model";
    }
}
?>

<!-- PHP -> ; are so important -->